pepper-apis/src/main/java/org/broadinstitute/ddp/route/ParticipantLookupByGuidRoute.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.ddp.route;
+
+import static org.apache.http.HttpStatus.SC_NOT_FOUND;
+import static org.broadinstitute.ddp.constants.ErrorCodes.USER_NOT_FOUND;
+import static org.broadinstitute.ddp.route.ParticipantsLookupRoute.handleParticipantLookupException;
```

Comment:
Feels a little weird to static import something from a route...

Reply:
Fixed this: moved common code to a separate util class.